docs/actions-reusable.md

```diff
@@ -37,6 +37,14 @@ The `config` property, which is optional, describes configuration options.
 
 ## Developing reusable actions
 
+!!! note
+    If you're thinking about developing a reusable action, then start by creating a new study within the [`opensafely`](https://github.com/opensafely) organisation that encapsulates the problem.
+    As a minimum, the study should [extract](actions-cohortextractor.md) and operate on a cohort:
+    indeed, the code that operates on the cohort *is* the reusable action.
+
+    At this point, you should [open an issue](https://github.com/opensafely-actions/.github/issues).
```

Comment:
question: If this is the contact route, is someone planning to watch/monitor this repository's issue? (The only repository watcher right now is Tom W.)

Reply:
Good point! I'm now watching for new issues and have also opened opensafely-actions/.github#1.